### PR TITLE
Re org debian docker

### PR DESCRIPTION
**Pull request by tejal29**
Hey all, 

This is new layout.
Most of the changes are moves and changing targets.

The new layout is now like this:
1. One directory per base image at top level

2. Each base image directory has a reproducible dir which contains all the scripts and definitions for building reproducible images.
     
3. All tests are now under tests/<base_image>/
    I moved a bunch of structure tests there.
Here is where you can browse through the dirs.
https://github.com/tejal29/base-images-docker/tree/re-org_debian_docer

### CODE REVIEW

**Comment by tejal29** [comment]
@dlorenc , @sharifelgamal , @aaron-prindle PTAL

File: .gcloudignore [hunk 1]
```diff
@@ -0,0 +1 @@
+./git/
```
Comment (dlorenc) [inline .gcloudignore]: I think we can drop this file, it's not actually used yet.

**Comment by tejal29** [comment]
I thought about that, but ubuntu is not really reproducible. The Dockerfile is outside the ubuntu/reproducible. 
I was thinking of removing them when switch to download_pkgs and install_pkgs.

I kept the node/reproducible for the same reason since it depends on ubuntu.
Can get rid of them in another PR.

**Comment by dlorenc** [comment]
> I kept the node/reproducible for the same reason since it depends on ubuntu.
> Can get rid of them in another PR.

SGTM.

**Pull request merged**